src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -548,10 +595,21 @@ function tryRemoveElement(elem) {
         /**
          * @private
          */
-        updateCurrentTrackOffset(offsetValue) {
+        updateCurrentTrackOffset(offsetValue, currentTrackIndex = 0) {
+            const skipRelativeOffset = currentTrackIndex !== PRIMARY_TEXT_TRACK_INDEX;
             let relativeOffset = offsetValue;
             const newTrackOffset = offsetValue;
-            if (this.#currentTrackOffset) {
+            if (this.#currentTrackOffset && !skipRelativeOffset) {
```

Comment:
This is a really great find! Thank you for pointing out this bug! You are correct, I was able to solve it by storing a separate value for secondary instead of skipping that relative offset calculation.